www/grpc/proto/blockchain.proto

```diff
@@ -298,8 +298,24 @@ message ConsensusInfo {
   int32 round = 4;
   // List of votes in the consensus instance.
   repeated VoteInfo votes = 5;
+  // The proposal of the consensus instance.
+  Proposal proposal = 6;
```

Comment:
All instances has the same proposal. So better we move it to `GetConsensusInfoResponse`